registry/registry.go

```diff
@@ -345,36 +345,30 @@ func configureReporting(app *handlers.App) http.Handler {
 func configureLogging(ctx context.Context, config *configuration.Configuration) (context.Context, error) {
 	logrus.SetLevel(logLevel(config.Log.Level))
 
-	formatter := config.Log.Formatter
-	if formatter == "" {
-		formatter = "text" // default formatter
-	}
-
-	switch formatter {
+	switch config.Log.Formatter {
 	case "json":
 		logrus.SetFormatter(&logrus.JSONFormatter{
 			TimestampFormat:   time.RFC3339Nano,
 			DisableHTMLEscape: true,
 		})
-	case "text":
+	case "text", "":
+		// Set the config to the default ("text") formatter in case none was
+		// specified.
+		config.Log.Formatter = "text"
```

Comment:
oh! Wait, I had my wires crossed when I wrote this. I thought the original code (before my change) updated the config struct, so I added this line.  I'm not a fan of updating the config itself, so let me move this back to the start (and use a local variable).